hotrod/src/main/java/io/hyperfoil/hotrod/config/HotRod.java

```diff
@@ -0,0 +1,22 @@
+package io.hyperfoil.hotrod.config;
+
+import java.io.Serializable;
+
+public class HotRod implements Serializable {
+
+   private final String uri;
```

Comment:
So when you want to run a test with two caches in the same cluster you need to duplicate the endpoint part. Why did you do so? What is the advantage to having a list of caches nested under the endpoint?

I am worried that once you start adding configuration options (besides the URI), e.g. number of connections... you'll run into the fact that single RCM is shared by multiple caches (based on URI) but you could have conflicting configuration for the RCM in endpoint+cacheName pairs.

Reply:
How about adding `cacheNames` as a list under `hotrod`.
Or how about about replace `cacheName` ( single element ) with `cacheNames`
?

Reply:
I would do it this way:
```yaml
hotrod:
- uri: hotrod://site1:11222
  caches:
  - cacheA
  - cacheB
  connectionPool: ...
- uri: hotrod://site2:11222
  caches: [ cacheA ]
```

Reply:
I didn't get why `connectionPool`. With the ISPN12 the `uri` is translated to `RCM`.

Example: `hotrod[s]://[username:password]@host[:port][,host[:port]…​][?property=value[&property=value…​]]`

Reply:
I meant `connectionPool` just as an example of additional configuration.
I didn't know that the properties can be passed in the query part; it would be nicer to have them exposed in a structured way as well.
If you use the URI as-is as the key in the map you'd probably have to repeat all the properties when setting `uri` for the individual request, which seems inconvenient and redundant.

Reply:
I agree but could we improve that in another PR?

Reply:
Sure, I didn't try to imply that all the config properties must have mapping right now.